hazelcast/src/hazelcast/client/compact.cpp

```diff
@@ -1495,6 +1497,7 @@ init_fp_table()
     }
     return FP_TABLE;
 }
+
 uint64_t
 FP_TABLE_AT(int index)
```

Comment:
I think that you may not need this extra method, if the fp_table is renamed as `fp_table` and it returns reference to the the static variable which should be initialized in a thread safe way.

Reply:
Fixed.